.github/workflows/lint.yaml

```diff
@@ -0,0 +1,17 @@
+name: lint
```

Comment:
Agreed. We should run this as part of pull requests / commits.

Reply:
Should be set up already I think: https://github.com/redhat-developer/vscode-xml/actions/runs/2861015002